workloads/templates/workload-fio-script-cm.yml.j2

```diff
@@ -0,0 +1,244 @@
+      for node in $master_nodes; do
+        echo "master" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+      infra_nodes=`oc get nodes -l pbench_agent=true,node-role.kubernetes.io/infra= --no-headers | awk '{print $1}'`
+      for node in $infra_nodes; do
+        echo "infra" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+      worker_nodes=`oc get nodes -l pbench_agent=true,node-role.kubernetes.io/worker= --no-headers | awk '{print $1}'`
+      for node in $worker_nodes; do
+        echo "worker" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+    fi
+    source /opt/pbench-agent/profile
+    echo "$(date -u) Done configuring pbench for FIO I/O scale test"
+    # End pbench Configuration
+    #
+    # Test Configuration
+    echo "$(date -u) Creating pods for FIO I/O scale test - pods will be in namespace:" {{ fiotest_basename }}0
+    pbench-user-benchmark --config="{{ fiotest_prefix }}-pods-{{ fiotest_maxpods }}-sc-{{ fiotest_storageclass }}-create_pods" -- 'VIPERCONFIG=/root/workload/fiotest.yml openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster should load the cluster [Suite:openshift]"'
+    echo "$(date -u) Pods for FIO I/O test created."
+
+    # wait until all pods are started and then collect data
+    while [[ $(oc get pods -n {{ fiotest_basename }}0 | grep fiotest-pod | grep Run |wc -l) -lt {{ fiotest_maxpods }}  ]] ; do
+      sleep 30
+      echo "Waiting on pods to start..."
+    done
+    ## necessary for user diffrent than root
+    clients=`oc get pods --output=json -n {{ fiotest_basename }}0 | jq -r '[.items[].status.podIP] | join(",")'`
+    mv ~/.ssh/config ~/.ssh/config.bak || true
+    for host in $(echo "${clients}" | sed "s/,/ /g");
+    do
+      HOST=${host} envsubst < /root/workload/ssh_config.template >> ~/.ssh/config
+    done
+    cat ~/.ssh/config.bak >> ~/.ssh/config || true
+    chmod 0600 ~/.ssh/config
+    cp /root/workload/fiotest.sh /tmp/
+
+    # Execute fio test
+    /tmp/fiotest.sh
+    pbench-copy-results --prefix "{{ fiotest_prefix }}"-"{{ fiotest_maxpods }}"-"{{ fiotest_storageclass }}"
+    if [[ {{ fiotest_cleanup }} == "true" ]]; then
+       oc delete project {{ fiotest_basename }}0
+       while [ "$(oc get project |grep {{ fiotest_basename }}0 | awk '{print $1}')" == {{ fiotest_basename }}0 ]; do
+         echo "waiting on project {{ fiotest_basename }}0 to disappear ..."
+         sleep 10
+       done
+       echo "Project {{ fiotest_basename }}0 is deleted ... test finished"
+    elif [[ {{ fiotest_cleanup }} == "false" ]]; then
+       echo "Test is done, but project {{ fiotest_basename }}0 is not be deleted due to FIOTEST_CLAENUP=false"
+    fi
+  fiotest.sh: |
+    #!/bin/bash
+    cp /root/.kube/config /tmp/config
+    export KUBECONFIG=/tmp/config
+    # Not sure is this necessary - re-test
+    sed -i 's/\-o StrictHostKeyChecking=no/\-o StrictHostKeyChecking=no \-l default \-p 2022/g' /opt/pbench-agent/base
```

Comment:
This line should be kept in the `pbench-agent.cfg` file on the image.  Why do we have to use a `sed` operation here to change pbench-agent code in the live image?